chamber/importers/__init__.py

```diff
@@ -43,19 +40,28 @@ class AbstractCSVImporter:
     delimiter = ';'
     encoding = 'utf-8'
 
-    def __call__(self, csv_path):
-        """csv_path is a required parameter as calling the function without CSV does not make sense"""
-        self.import_csv(custom_csv_path=csv_path)
-
-    def import_csv(self, custom_csv_path=None):
-        with io.open(self.get_filename(custom_csv_path=custom_csv_path), encoding=self.get_encoding()) as f:
-            reader = csv.reader(f, delimiter=self.get_delimiter())
-            if self.get_skip_header():
-                next(reader, None)
-            self.import_rows(
-                reader,
-                row_count=simple_count(self.get_filename(custom_csv_path=custom_csv_path), encoding=self.get_encoding())
-            )
+    def __call__(self, file):
+        """file is a required parameter as calling the function without CSV file does not make sense"""
+        self.import_csv(file)
+
+    def import_csv(self, file=None):
+        if not file:
+            with open(self.csv_path, encoding=self.get_encoding()) as file:
```

Comment:
@matllubos Is loading from disk really a needed feature or should we just drop it entirely and keep only the file-like object way.